src/sentry/integrations/vercel/integration.py

```diff
@@ -150,6 +157,35 @@ def build_integration(self, state):
                 "installation_type": installation_type,
                 "webhook_id": webhook["id"],
             },
+            "post_install_data": {"user_id": state["user_id"]},
         }
 
         return integration
+
+    def post_install(self, integration, organization, extra=None):
+        # check if we have an installation already
+        if SentryAppInstallationForProvider.objects.filter(
+            organization=organization, provider="vercel"
+        ).exists():
+            logger.info(
+                "vercel.post_install.installation_exists",
+                extra={"organization_id": organization.id},
+            )
+            return
+
+        user = User.objects.get(id=extra.get("user_id"))
+        data = {
```

Comment:
I think you'll have to add this to the mediator - but can we add the author for these internal integrations to be "Sentry-Vercel Integration" or something? so that we can display 'authored by' in the UI for developer settings? I know that we are setting the overview but I think we could have both

Reply:
@MeredithAnya My original plan was to set the author. The only problem is that we actually don't display the author field for internal integrations. So we'd have to make a UI change on the developer settings page to enable this.

Reply:
I think its worth adding in the UI since I think it has come up before that it's hard to know who created the internal integration. This could also be done in a follow up, but maybe a comment could be good to remind us add it in later whenever we do the UI change

Reply:
@MeredithAnya The UI change shouldn't be hard, I can add it here.

Reply:
@MeredithAnya Upon further thought, I think the change should be done in a follow-up PR. We wanted to actually show the author as the user who made the integration instead of just a form field. Let me add a comment and make a card for this.

Reply:
Made the change so we can set the author when we create the internal integration for Vercel but not so you can do it in the UI. We'll make that change later.